README.md

```diff
@@ -63,7 +63,7 @@ You can visit [doc page](https://hugegraph.apache.org/docs/quickstart/hugegraph-
 
 ## License
 
-HugeGraph is licensed under Apache 2.0 License.
+HugeGraph is licensed under [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0).
```

Comment:
Thanks but maybe use our current file is better?

https://github.com/apache/incubator-hugegraph/blob/master/LICENSE

u could use abs path or relative path for it (both fine)

Reply:
thank you, done